src/modules/content-tree/content.tree.module.js

```diff
@@ -14,7 +14,7 @@ export default class ContentTreeModule extends Component {
         this._refTreeContainer = React.createRef();
 
         this.state = {
-            selectedLocationId: props.selectedLocationId,
+            currentLocationId: props.currentLocationId,
```

Comment:
We need to store it in _state_ because in future it will be changing during lifetime of component?